src/main/java/net/minecraftforge/fml/common/langsupport/IPropertyWrapper.java

```diff
@@ -0,0 +1,61 @@
+package net.minecraftforge.fml.common.langsupport;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Field;
+
+public interface IPropertyWrapper
+{
+    String getName();
+
+    Class<?> getType();
+
+    void set(Object value, boolean ignoreAccess) throws IllegalAccessException;
+
+    void get() throws IllegalAccessException;
+
+    boolean isAnnotationPresent(Class<? extends Annotation> annotationClass);
+
+    class FieldWrapper implements IPropertyWrapper
+    {
+        private Object instance;
+        private final Field field;
+
+        public FieldWrapper(Field field, Object instance)
+        {
+            this.field = field;
+            this.instance = instance;
+        }
+
+        @Override
+        public String getName()
+        {
+            return field.getName();
+        }
+
+        @Override
+        public Class<?> getType()
+        {
+            return field.getType();
+        }
+
+        @Override
+        public void set(Object value, boolean ignoreAccess) throws IllegalAccessException
+        {
+            if (ignoreAccess)
+                field.setAccessible(true);
```

Comment:
This will ignore access from now on until the end of time, you might want to reset the flag afterwards, or just always call `setAccessible` with the value of `ignoreAccess`.